### PR TITLE
Fix font

### PR DESCRIPTION
**Pull request by mrbwaters**
Fixed font sizing issue.
Created a spr_textbox backdrop for text.
Created dynamically aligned text.
Added faces to character sprites for fun.

Will resolve merge conflicts here.

Next step:
@jbussing I think we need a Talking pc state - let's discuss what that looks like.
Let's think about the best way to organize and represent dialog. This way works but it's not super scalable.

### CODE REVIEW

**Comment by mrbwaters** [comment]
Conflicts resolved (I hope smoothly 😬 )

**Comment by jbussing** [comment]
Would talking be equivalent to interact.. like just a frozen pc or something different. Im down for either, but if we can do more with less im down with that too.

Also ping me when you want to push this 🚀

**Comment by jbussing** [comment]
.. And would that be just talking state for pc or both pc and npc?

**Comment by mrbwaters** [comment]
I don't know what the INTERACT state might look like yet or even if we need one. I think we need a meeting of the minds to clarify exactly what that state would be. Likely TALK and INTERACT could be the same thing.

I'm imagining the TALK state to pause movement, create the textbox, maybe change animation.

If we have an FSM for NPCs then I'd imagine this would be a state for both of them, yeah. Though most NPCs probably only have two states — whatever they do when the PC doesn't talk to them and talk to the PC. So at least it's a simple FSM 😅

We should also think conceptually about things that affect the entire game, like PAUSE. Is that a pc state or should there also be an FSM for the meta_game (running, pause, etc.)

Long story short: merge this PR when you're ready. And playtest the game! I'm pretty pleased with the little dialog system here 🎉

Sent via Superhuman ( ***@***.*** )

**Comment by jbussing** [comment]
I think we can sync later on the Pause state... Im looking forward to play testing the game!

Also, I just had thought as Im writing this. We should have the other group members play-test the games from the prior "Climb the Tower" challenge. Do those games even have a pause feature? Im trying to get at a feel for what the competition looks like and what we might want to minimally plan for in the future. We should talk about how strategically task our other group members with this research. Things they like, things they don't like, common features, trends, and etc.

I think it's good that we know how to pause from a technical learning perspective, but Im also curious to know about what other submissions look like and if that might even inform our scope and planning.

**Comment by jbussing** [comment]
also I see you with the PNG and JPG looking forward to the new dialogue! 🚢
